S23/fulvius31/app/src/main/java/lowercase/Lowercaser.java

```diff
@@ -0,0 +1,115 @@
+package lowercase;
+
+import java.text.Normalizer;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class Lowercaser {
+
+    private final String upperCase;
```

Comment:
Slightly unusual design here; I might have guessed that a "Lowercaser" class represented just the "lowercasing algorithm", in which case you'd only want to pass the language code to the constructor, and then pass the string you want to lowercase as an argument to a "lowercase" method.